src/test/java/edu/ucla/library/bucketeer/verticles/ClearCacheVerticleTest.java

```diff
@@ -0,0 +1,153 @@
+     */
+    private String myVertID;
+
+    /**
+     * Sets up the tests.
+     *
+     * @param aContext A testing environment
+     */
+    @Before
+    public final void setup(final TestContext aContext) {
+        final ConfigRetriever configRetriever = ConfigRetriever.create(Vertx.vertx());
+        final Async asyncTask = aContext.async();
+
+        configRetriever.getConfig(configuration -> {
+            if (configuration.failed()) {
+                aContext.fail(configuration.cause());
+            } else {
+                final JsonObject config = configuration.result();
+                final DeploymentOptions options = new DeploymentOptions().setConfig(config);
+
+                myRunTestOnContextRule.vertx()
+                    .deployVerticle(ClearCacheVerticle.class.getName(), options, deployment -> {
+                        if (deployment.succeeded()) {
+                            myVertID = deployment.result();
+                            TestUtils.complete(asyncTask);
+                        } else {
+                            aContext.fail(deployment.cause());
+                        }
+
+                    });
+            }
+        });
+
+
+    }
+
+
+    /**
+     * Tear down the testing environment.
+     *
+     * @param aContext A test context
+     */
+    @After
+    public void tearDown(final TestContext aContext) {
+
+        if (myVertID != null) {
+            final Async async = aContext.async();
+            myRunTestOnContextRule.vertx().undeploy(myVertID, undeployment -> {
+                if (undeployment.failed()) {
+                    aContext.fail(undeployment.cause());
+                } else {
+                    async.complete();
+                }
+            });
+        } else {
+            LOGGER.info(MessageCodes.BUCKETEER_605);
+        }
+
+
+
+       //in tear down get verticle id and shut down the verticle by referencing the ID
+    }
+
+    /**
+     * Tests clearing Cantaloupe's cache.
+     *
+     * @param aContext A testing context
+     */
+    @Test
+    public void testCacheClear(final TestContext aContext) {
+
+        final WebClient client = WebClient.create(Vertx.vertx());
+
+        // These are just the property names, not values
+        LOGGER.info(MessageCodes.BUCKETEER_606, StringUtils.trimToNull(myUsername));
+
+        //Sends multiple messges over eventBus to see if correct response is recieved each time
+        for (int i = 0; i < 500; i++) {
+            final Async asyncTask = aContext.async();
+            myRunTestOnContextRule.vertx()
+                .eventBus().<JsonObject>send(ClearCacheVerticle.class.getName(), new JsonObject()
+                .put("imageID", "ARK-12345678"), response -> {
+                    if (response.failed()) {
+                        aContext.fail(response.cause());
+                    } else {
+                        TestUtils.complete(asyncTask);
+                    }
+                });
+            i++;
```

Comment:
Do you mean this to be double incremented? I don't _think_ so but, if so, it might be better to do `index += 2` in the `for` above?